### ISSUE TITLE
Math block / inline math very blurry

### ISSUE THREAD

**Issue by fengyuli2002** [typora/typora-issues#5441]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [x] Searched http://support.typora.io/

----

**Describe the bug**
Inline math and math blocks are very very blurry.

**To Reproduce**
I tried reinstall. Just download typora and type some math.

**Screenshots / Screencasts**
<img width="322" alt="image" src="https://user-images.githubusercontent.com/76980494/191621855-05567257-b346-4c1a-8f5e-00cb3968fc7c.png">
<img width="318" alt="image" src="https://user-images.githubusercontent.com/76980494/191621914-0d107cb5-5e79-4893-8fb0-46cd30979e18.png">

**Sample Markdown File**

**Desktop (please complete the following information):**
 - OS: MacOS Ventura

**Typora Version**
e.g: 1.4.4

**Additional context**

**Comment by abnerlee** [comment]
Do you use default theme without any custom CSS ?
Does reboot macOs fix this?
Could you provide the sample md file?

**Comment by Laplaceeee** [comment]
![Screenshot 2022-10-20 at 11 09 34 PM](https://user-images.githubusercontent.com/38410778/197102499-8c3d4f89-0009-494d-9037-ce1ef36d1bac.png)

I have the same issue.

**Desktop:**
OS: MacOS Ventura

**Typora Version:**
1.4.4 and 1.3.7

**Comment by WonderLandxD** [comment]
> 

I have the same issue, too. When using the macOS Monterey, the Math block / inline math shows up perfectly. After pushing macOS Ventura today, the display became very blurry. But after exporting to pdf, the formula in pdf is displayed normally. I also tried rebooting the computer and changing the version, but it didn't work. Guessing it's probably due to system adaptation.

**Desktop:**
OS: macOS Ventura

**Typora Version:**
Stable Releases: 1.4.4 and 1.3.7
Dev Releases: 1.4.1
[Test for Math.md](https://github.com/typora/typora-issues/files/9859432/Test.for.Math.md)

<img width="956" alt="test" src="https://user-images.githubusercontent.com/100405945/197749490-08f45280-a46a-4033-845c-d2c8114abe64.png">

**Comment by yzy1996** [comment]
Any progress?

**Comment by otzslayer** [comment]
I found a temporary solution. The key of this problem is in the below path.
```
/Applications/Typora.app/Contents/Resources/TypeMark/lib/MathJax3/es5/tex-svg-full.js
```
I replaced all jax="SVG" with jax="CHTML" in this file and then re-run Typora.
Thanks for your insight! @Laplaceeee

**Comment by WonderLandxD** [comment]
> 

It works! Thanks for your solution! @otzslayer @Laplaceeee

**Comment by LiangLee17** [comment]
> I found a temporary solution. The key of this problem is in the below path.
> 
> ```
> /Applications/Typora.app/Contents/Resources/TypeMark/lib/MathJax3/es5/tex-svg-full.js
> ```
> 
> I replaced all jax="SVG" with jax="CHTML" in this file and then re-run Typora. Thanks for your insight! @Laplaceeee

Thank you! I have bothered by this a day long. It works!

**Comment by abnerlee** [comment]
fixed in v1.4.8
